monkey-runner/README.md

```diff
@@ -41,6 +44,7 @@ buildscript {
 
 ...
 
+apply plugin: 'com.novoda.android-command'
```

Comment:
Should we add the `classpath` above for the command plugin as well?
Because it won't work without it.